Zend/zend_execute_API.c

```diff
@@ -851,6 +851,9 @@ int zend_call_function(zend_fcall_info *fci, zend_fcall_info_cache *fci_cache) /
 		if (EXPECTED((func->op_array.fn_flags & ZEND_ACC_GENERATOR) == 0)) {
 			zend_init_execute_data(call, &func->op_array, fci->retval);
 			zend_execute_ex(call);
+			if (UNEXPECTED(ZEND_CALL_INFO(call) & ZEND_CALL_CLOSURE)) {
+        			OBJ_RELEASE((zend_object*)func->op_array.prototype);
+			}
```

Comment:
Right, it's that memory leak. We already have that leak before this patch… I actually could have committed this independently.

Whether it'll slowdown, don't know. This is just for zend_call_function… and in ZEND_DO_FCALL, branch predictor usually will be correct… it's unexpected after all… so maybe three ops. But yeah, it's necessary.